### PR TITLE
UI for gas station. Part 1/3 EVM gas

### PR DESCRIPTION
**Pull request by niklabh**
UI preview: https://gas.astar.network/

Screenshot from 2023-03-20 14-09-54

This PR setup Create react app for the UI. This UI adding EVM gas and EIP1559 gas values like base fee and priority fee.
Will do 2 another PRs for extrinsic tips and contract weights

### CODE REVIEW

File: src/GasList.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+        setTransactions(result);
+      }
+    }).catch(error => {
+      setError(error);
+    });
+  }, [network]);
+
+  return (
+    <React.Fragment>
```
Comment (impelcrypto) [inline src/GasList.tsx]: You can also use `<> </>` instead of <React.Fragment>

**Comment by impelcrypto** [comment]
Let's fix styling for mobile devices

**Review by Kahonnohak — DISMISSED** [review]
LGTM

**Comment by Kahonnohak** [comment]
It's better to hide categories you don't have yet? Such as Tips?

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (bobo-k2) [inline public/manifest.json]: Maybe to update short_name, name. Also, update logo images

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/GasList.tsx (hunk 1 above)
Comment (niklabh) [reply src/GasList.tsx]: removed

**Comment by niklabh** [comment]
> Let's fix styling for mobile devices
> 
> 

Removed on mobile

File: public/manifest.json (hunk 2 above)
Comment (niklabh) [reply public/manifest.json]: done

**Review by impelcrypto — APPROVED** [review]

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Comment by Kahonnohak** [comment]
Hi @niklabh since you added the mobile design, one fix on the header please. And let me confirm if we want Gwei or ASTR
スクリーンショット 2023-03-27 11 02 35

**Comment by niklabh** [comment]
> Hi @niklabh since you added the mobile design, one fix on the header please. And let me confirm if we want Gwei or ASTR スクリーンショット 2023-03-27 11 02 35

Fixing in next PR

**Pull request merged**